install.py

```diff
@@ -151,9 +181,9 @@ def install(rc_path=None, bin_path=None, force=False):
     fp.write('\n' + alias_comment + '\n')
     fp.write(alias_cmd)
     fp.close()
-    hai('{blue}[✓]{/blue} {green}Created "grow" alias in:{/green} %s' % rc_path)
+    hai('{blue}[✓]{/blue} {green}Created "grow" alias in:{/green} {}', rc_path)
     hai('{green}All done. To use Grow SDK...{/green}')
-    hai(' ...reload your shell session OR use `source {}`,'.format(rc_path))
+    hai(' ...reload your shell session OR use `source {}`,', rc_path)
```

Comment:
The new gcloud SDK has the user do this to reload their shell instead:
```
exec -l $SHELL
```

WDYT of that approach, instead of telling people to resource a file? (Those instructions came from here - https://cloud.google.com/sdk/downloads#mac)